examples/multi-layer/square_multi/Makefile

```diff
@@ -0,0 +1,124 @@
+
```

Comment:
Containing directory is misspelled.  I might suggest just shortening it to `square` or `square_basin`.